integration-tests/experimental/callbacks/test_llama_index_callback_handler.py

```diff
@@ -68,6 +70,13 @@ def test_callback_handler_records_llm_message_attributes_for_query_engine(
     assert role == "user"
     assert query in message_text
 
+    span = next(span for span in tracer.span_buffer if span.span_kind == SpanKind.EMBEDDING)
+    embedding_texts = span.attributes["embedding.text"]
+    embedding_vectors = span.attributes["embedding.vector"]
```

Comment:
are these plural or singular?
```suggestion
    embedding_texts = span.attributes[EMBEDDING_TEXT]
    embedding_vectors = span.attributes[EMBEDDING_VECTOR]
```

Reply:
addressed in [2fe71e5](https://github.com/Arize-ai/phoenix/pull/1169/commits/2fe71e55efc00378f6c7b82fa6e08905b0348bfb)